frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -1,25 +1,43 @@
 import React, { useEffect, useState } from "react";
-import { getSightings, Sighting } from "../../clients/apiClient";
+import {
+  getSightings,
+  getSightingsBySpeciesId,
+  SightingWithLocations,
+} from "../../clients/apiClient";
 import { SightingCard } from "./SightingCard";
 import "./BrowseSightings.scss";
+import { useParams } from "react-router-dom";
 
 export const BrowseSightings: React.FC = () => {
-  const [sightings, setSightings] = useState<Sighting[]>();
+  const [sightings, setSightings] = useState<SightingWithLocations[]>();
+  const { speciesId } = useParams<{ speciesId: string }>();
 
-  useEffect(() => {
-    getSightings().then(setSightings);
-  }, []);
+  if (speciesId === undefined) {
+    useEffect(() => {
+      getSightings().then(setSightings);
+    }, []);
+  } else {
+    useEffect(() => {
+      getSightingsBySpeciesId(speciesId).then(setSightings);
+    }, [speciesId]);
+  }
+
+  let pageHeadline = "";
 
   if (sightings === undefined) {
-    return <p>Loading</p>;
+    pageHeadline = "Loading";
+  } else if (sightings.length === 0) {
+    pageHeadline = "Sorry, no whales of that species have been seen";
+  } else {
+    pageHeadline = "Reported Sightings";
   }
 
   return (
     <>
-      <h1>Reported Sightings</h1>
+      <h1>{pageHeadline}</h1>
       <ul>
```

Comment:
This should be the conditional part:
```suggestion
      {message === "" ? <ul>
```
where `message` is what you defined earlier, either "Loading..." or "No whales of that species have been seen" (or "No sightings to display" or whatever)

Then, below,
```
</ul>
: <p>{message}</p>}
```

Reply:
done, but I think i've done it in a slightly different way, hopefully you still think it's ok!

Reply:
done, but I think i've done it in a slightly different way, hopefully you still think it's ok!